shared/versions.yml

```diff
@@ -70,18 +70,34 @@ couchdb:
 dataloader:
   upstream:
     repository: gpii/universal
-    tag: 20190522142238-4a52f56
+    tag: 20190610144430-9bd021f
   generated:
-    repository: gcr.io/gpii-common-prd/gpii/universal
-    sha: sha256:35eafa2a5fe1e5b4dd908596854fd46015a75c4f0758c743c2491868fe8b12fa
-    tag: 20190522142238-4a52f56
+    repository: gpii/universal
+    sha: sha256:42f238d64e3afec834a1da2b33b602be68e6acb437d623b2b5c559c61a9f6b2a
+    tag: 20190610144430-9bd021f
+flushtokens:
+  upstream:
+    repository: gpii/universal
+    tag: 20190610144430-9bd021f
+  generated:
+    repository: gpii/universal
+    sha: sha256:42f238d64e3afec834a1da2b33b602be68e6acb437d623b2b5c559c61a9f6b2a
+    tag: 20190610144430-9bd021f
+istio_proxy_manager:
+  upstream:
+    repository: gcr.io/gke-release/istio/proxyv2
+    tag: 1.0.6-gke.3
+  generated:
+    repository: gcr.io/gke-release/istio/proxyv2
+    sha: sha256:1f6dac3dbf69c0b0d519398003b6572e0218f695f1be295a2ffff95df2395ed0
+    tag: 1.0.6-gke.3
 flowmanager:
   upstream:
     repository: gpii/universal
     tag: 20190522142238-4a52f56
   generated:
-    repository: gcr.io/gpii-common-prd/gpii/universal
-    sha: sha256:35eafa2a5fe1e5b4dd908596854fd46015a75c4f0758c743c2491868fe8b12fa
+    repository: gpii/universal
```

Comment:
These changes probably should not be here?

Reply:
I followed the ["Running on host" directions](https://github.com/gpii-ops/gpii-version-updater#running-on-host) in the `gpii-version-updater` README.  The `generated` section is created by executing `rake sync`.

I think when the `upstream` section of `versions.yml` is set up correctly, as discussed above, then the `generated` section should be fine.